src/style.css

```
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+  display: block;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  padding: 0;
+
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
+}
```

```suggestion
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```
better to make bargin from one side.
and with such selector will be settled margin for all elements except last.